modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -6,17 +6,17 @@
 [id="updating-eus-to-eus-upgrade_{context}"]
 = EUS-to-EUS update
 
-The following procedure pauses all non-master MachineConfigPools and performs updates from {product-title} 4.8 to 4.9 to 4.10, then unpauses the previously paused MachineConfigPools.
+The following procedure pauses all non-master MachineConfigPools and performs updates from {product-title} 4.10 to 4.11 to 4.12, then unpauses the previously paused MachineConfigPools.
 Following this procedure reduces the total update duration and the number of times worker nodes are restarted.
 
 .Prerequisites
 
-* Review the release notes for {product-title} 4.9 and 4.10
+* Review the release notes for {product-title} 4.11 and 4.12
 * Review the release notes and product lifecycles for any layered products and OLM Operators. Some may require updates either before or during an EUS-to-EUS update.
-* Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to updating from {product-title} 4.8 to 4.9.
-* Verify that your cluster is running {product-title} version 4.8.14 or later.
-If your cluster is running a version earlier than {product-title} 4.8.14, you must update to a later 4.8.z version before updating to 4.9.
-The update to 4.8.14 or later is necessary to fulfill the minimum version requirements that must be performed without pausing MachineConfigPools.
+* Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.11/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to updating from {product-title} 4.10 to 4.11.
+* Verify that your cluster is running {product-title} version 4.10.14 or later.
```

Comment:
We can drop the 4.8.14 mentions entirely.  They had been added to cover [the addition of admin-ack guards][1], ensuring that folks had those in place before moving to 4.9.  But  with 4.10 and later, we have conditional update risk declarations, and there are no supported updates that violate our current build recommendations.  In [the graph-data repo][2]:

```console
$ grep minor_min build-suggestions/4.1[12].yaml
build-suggestions/4.11.yaml:  minor_min: 4.10.16
build-suggestions/4.12.yaml:  minor_min: 4.11.11
$ hack/show-edges.py --cincinnati https://api.openshift.com/api/upgrades_info/graph stable-4.11 | grep '^4[.]10[.]\([0-9]\|1[0-5]\) -.* 4[.]11[.]'
...no hits...
$ hack/show-edges.py --cincinnati https://api.openshift.com/api/upgrades_info/graph fast-4.12 | grep '^4[.]11[.]\([0-9]\|10\) -.* 4[.]12[.]'
...no hits...
```

And if we discover update risks where dangling compute nodes or something might endanger an update, we would address that by declaring conditional update risks via api.openshift.com, and not by hoping that customers find and evaluate update risks declared only in these docs.

[1]: https://github.com/openshift/cincinnati-graph-data/commit/2b20f927ccd7c90718a070170c64d2f61dc72ac1
[2]: https://github.com/openshift/cincinnati-graph-data/tree/2fbdc578fafb06c732fe1768f3a6ea8fbfe8eec7